tests/iso-8859-2-hungarian/auto-apro.hu.xml

```diff
@@ -1,474 +0,0 @@
-<?xml version="1.0" encoding="iso-8859-2" ?>
```

Comment:
The tag removing & filtering (with nested tags too) with using regex (and without it) I've implemented in some my projects, but here we need also some simple heuristic, because some XML and HTML files contains something like:
![img](https://cloud.githubusercontent.com/assets/10864177/6234021/89a97ab0-b6d7-11e4-9c0e-b48d3a7458b6.png)